client/my-sites/plans/ecommerce-trial/trial-feature-card/style.scss

```diff
@@ -13,87 +14,108 @@
-		width: 115px;
-		height: 65px;
-
-		@media (max-width: $break-mobile) {
-			display: none;
-		}
-	}
-
-	.trial-feature-card__content {
-		flex: 1;
+	.trial-feature-card__header {
 		display: flex;
-		flex-direction: column;
-		padding: 5px 25px;
+		padding: 24px;
+		cursor: pointer;
+		text-align: left;
+		line-height: 1.5;
 
 		@media (max-width: $break-mobile) {
 			padding: 0;
 		}
 
+		.trial-feature-card__illustration {
+			width: 115px;
+			height: 65px;
 
-		.trial-feature-card__content-title {
-			font-size: $font-title-small;
-			font-weight: 500;
+			@media (max-width: $break-mobile) {
+				display: none;
+			}
 		}
 
-		.trial-feature-card__content-subtitle {
-			font-size: $font-body;
-			color: var(--color-neutral-40);
+		.trial-feature-card__header-text {
+			flex: 1;
+			padding: 0 24px;
 
 			@media (max-width: $break-mobile) {
-				display: none;
+				padding: 0;
+			}
+
+			.trial-feature-card__title {
+				font-size: $font-title-small;
+				font-weight: 500;
+			}
+
+			.trial-feature-card__subtitle {
+				font-size: $font-body;
+				color: var(--color-neutral-40);
+				font-weight: 400;
+
+				@media (max-width: $break-mobile) {
+					display: none;
+				}
 			}
 		}
 
-		.trial-feature-card__content-items {
-			margin-top: 45px;
+		.trial-feature-card__accordion-toggle {
+			cursor: pointer;
+			margin-top: 25px;
 
 			@media (max-width: $break-mobile) {
-				margin-top: 30px;
+				margin-top: 10px;
 			}
 
-			.trial-feature-card__content-items-item {
-				display: flex;
-				padding: 7px 0;
+			.gridicons-chevron-down,
+			.gridicons-chevron-up {
+				fill: var(--color-neutral-90);
+			}
+		}
+	}
 
-				.trial-feature-card__content-items-item-icon {
-					padding: 3px 0;
+	.trial-feature-card__items {
+		flex: 1;
+		display: flex;
+		flex-direction: column;
+		max-width: 590px;
+		margin-left: 140px;
+		// Special padding-top to match designs
+		padding: 10px 24px 24px 24px;
 
-					.gridicons-checkmark {
-						fill: var(--color-success-40);
-					}
-				}
 
-				.trial-feature-card__content-items-item-content {
-					flex: 1;
-					display: flex;
-					flex-direction: column;
-					margin-left: 20px;
+		@media (max-width: $break-mobile) {
+			padding: 0;
+			margin-top: 30px;
+			margin-left: 0;
+		}
 
-					.trial-feature-card__content-items-item-content-title {
-						font-size: $font-body;
-					}
+		.trial-feature-card__item {
+			display: flex;
+			padding: 7px 0;
 
-					.trial-feature-card__content-items-item-content-subtitle {
-						font-size: $font-body-extra-small;
-						color: var(--color-neutral-40);
-					}
+			.trial-feature-card__item-icon {
+				padding: 3px 0;
 
+				.gridicons-checkmark {
+					fill: var(--color-success-40);
 				}
 			}
-		}
-	}
 
-	.trial-feature-card__accordion-toggle {
-		display: inline-flex;
-		cursor: pointer;
-		padding: 10px;
+			.trial-feature-card__item-content {
+				flex: 1;
+				display: flex;
+				flex-direction: column;
+				margin-left: 20px;
```

Comment:
For mobile screens, I think we should add a rule to either reduce this to 16px to match the right padding, or increase the padding on the right of these items by 4px.

As things stand, the whitespace between the check mark and the text is 20px, but the space between the text and the right edge is 16px, and it looks a bit unbalanced to me. (See screenshot below.)

<img width="415" alt="Screenshot 2023-02-09 at 22 35 08" src="https://user-images.githubusercontent.com/3376401/217933157-559e8f74-bee2-4aa3-821c-32d77af30151.png">